compiler/packages/babel-plugin-react-compiler/src/Inference/InferEffectDependencies.ts

```diff
@@ -89,9 +90,33 @@ export function inferEffectDependencies(fn: HIRFunction): void {
           lvalue.identifier.id,
           instr as TInstruction<FunctionExpression>,
         );
+      } else if (
+        value.kind === 'PropertyLoad' &&
+        typeof value.property === 'string' &&
+        autodepModuleLoads.has(value.object.identifier.id)
```

Comment:
codestyle nit (feel free to take or leave it) -- I find it's easier to read long nested if-else chains if the outer / inner ifs are simple. e.g.
```
for (const instr of block.instructions) {
  if (value.kind === "FunctionExpression") {
    ...
  } else if (value.kind === "PropertyLoad") {
    const moduleTarget = autodepModuleLoads.get(
      value.object.identifier.id,
    );
    if (...) {
      ...
    }
  } else if (value.kind === "CallExpression" || value.kind === "MethodCall") {
    ...
  } 
}
```

Reply:
That's reasonable! I see why that would be easier to read